### DISCUSSION TITLE
Flatpak Package Names with Nix-Flatpak

### DISCUSSION THREAD

**Discussion by guttermonk** [gmodena/nix-flatpak#75] [Q&A] [answered]
Everything appears to be working well, but when I go to the menu and search for [ProtonUp-Qt](https://flathub.org/apps/net.davidotek.pupgui2) the app gets filtered out. If I search for pupgui2 (the name it has listed in manual install instructions on flathub) in my menu, it shows up and is listed as ProtonUp-Qt.

Before I switched to nix-flatpak, I had installed the ProtonUp-Qt Flatpak manually (not declaratively) and was able to search for the app in my menu by its name, like normal. Not sure if this module does something to the package name, in regards to the menu. Any thoughts or suggestions?

**Comment by gmodena** [comment]
hey @guttermonk . This module simply wraps the `flatplak` CLI, and I'd expect its behavior to be consistent to manual installations. It should not interfere with your desktop manager.

- what desktop manager do you use? FWIW Gnome is the one I've battle tested the most.
- did you previously install the package at user of system level? 
- do you use nix-flapak as a nixos module or via home-manager?

**Comment by guttermonk** [comment]
Hi @gmodena, thanks for writing me back. I'm using Hyprland. One thing I just noticed is that it only does this using Wofi. When I use Rofi, it works as expected.

Here's the script that I use to launch my menu typically:

```
#!/usr/bin/env bash

CONFIG="$HOME/.config/hypr/wofifull/config"
STYLE="$HOME/.config/hypr/wofifull/style.css"
COLORS="$HOME/.config/hypr/wofifull/colors"

if [[ ! $(pidof wofi) ]]; then
	wofi --show drun --prompt 'Search...' --conf ${CONFIG} --style ${STYLE} --color ${COLORS}
else
	pkill wofi
fi

```
Seems odd that Wofi would handle it differently than Rofi. According to [this post](https://todo.sr.ht/~scoopta/wofi/202#event-245552), Wofi should search for the "Name" field first. So you'd think it would find ProtonUp-Qt based on the flatpak's .desktop entry:

```
[Desktop Entry]
Name=ProtonUp-Qt
Exec=net.davidotek.pupgui2
Type=Application
Comment=Install Wine and Proton-based Compatibility Tools
Comment[de]=Wine und Proton Kompatibilitätstools installieren
Comment[it]=Installa tool di compatibilità basati su Wine e Proton
Comment[nl]=Installeer op Wine en Proton gebaseerde compatibiliteitshulpmiddelen
Comment[pl]=Zainstaluj narzędzia kompatybilności oparte na Wine i Protonie
Terminal=false
Icon=net.davidotek.pupgui2
Categories=Game;Utility;
```

**Comment by guttermonk** [comment, marked as answer]
I found the culprit. In my Wofi config, I had "insensitive=false" ...flipping this to true fixed the problem.

Hope this helps if anyone else has the same problem.

**Comment by gmodena** [comment]
@guttermonk great to hear you fixed it! Thanks for reporting and following up.
